uom-plugin/uom-plugin.cabal

```diff
@@ -2,7 +2,7 @@
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: d110f2f74fb66155437b924f27283bd9b16991b4f2be45693731488d98f4938c
```

Comment:
@philderbeast what is this a hash of, and why does it keep changing? It seems like this is likely to introduce unnecessarily conflicts.